### PR TITLE
Add TF EfficientNet V2 to benchmarks

### PR DESCRIPTION
**Pull request by mariecwhite**
This tests depthwise convolutions

### CODE REVIEW

**Comment by GMNGeoffrey** [comment]
Let's just watch out for the time it takes to build and run benchmarks. We've added quite a few recently and I'm seeing the time creep up. We may want to shard across more machines at some point

File: benchmarks/TF/CMakeLists.txt [hunk 1]
```diff
@@ -58,6 +58,20 @@ set(BERT_FOR_MASKED_LM_FP32_SEQLEN512_MODULE
     "1x512xi32,1x512xi32"
 )
 
+set(EFFICIENTNET_V2_S_TF_FP32_MODULE
+  NAME
+    "EfficientNetV2STf"
```
[the remark this answers is not in the document]
Comment (mariecwhite) [reply benchmarks/TF/CMakeLists.txt:63]: Done

**Comment by mariecwhite** [comment]
> 

Will do. Thanks for the example.

**Comment by mariecwhite** [comment]
> Since we are migrating the benchmark suites. Would you mind also adding the new model to the new benchmark suites?
> 
> I'm still working on docs, but here is an example: #11172

Will do.

**Pull request merged**